src/components/Counter/index.js

```diff
@@ -0,0 +1,3 @@
+import Counter from "./Counter";
+
+export { Counter };
```

Comment:
We can also write this as
`
export default Counter
`

Reply:
Fixed for all components (b0cb3c88d572bc6a2671a7f5afb18272e51cc346)